src/app/features/tech-record/components/vehicle-technical-record/vehicle-technical-record.component.html

```diff
@@ -13,6 +13,15 @@
           (submitChange)="handleSubmit()"
         ></app-edit-tech-record-button>
 
+        <ng-container
```

Comment:
Need to add role based access here though to Tech.Amend

Reply:
This will hide the button

Reply:
I was running into trouble with having both the ngIf and role required being in the container, so I put the role required instead in the button, does that work?

Reply:
check tech-record.component,html at the top there for an example where the same logic is applied :)